sdk/cxx/tp_intkey_tester.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2016 Intel Corporation
```

Comment:
This file shouldn't be needed, since it already exists at `sdk/examples/intkey_python/tests/test_tp_intkey.py`